example/storybook/stories/Notifications/index.tsx

```diff
@@ -198,7 +146,7 @@ export const NotificationsScreen = () => {
         </TouchableOpacity>
       </View>
 
-      {events.map((event, idx) => (
+      {(events as unknown as Event[]).map((event: Event, idx: number) => (
```

Comment:
this is ugly and fallout from some typing a bit higher in the app tree. will detail it below in this PR